extra/sigmund~/sigmund~.c

```diff
@@ -838,7 +838,7 @@ typedef struct _sigmund
     int x_npts;         /* number of points in analysis window */
     int x_npeak;        /* number of peaks to find */
     int x_loud;         /* debug level */
-    t_sample *x_inbuf;  /* input buffer */
+    t_float *x_inbuf;  /* input buffer */
```

Comment:
this doesn't look right.
i think it should be done the other way around (more `t_sample` rather than less)

Reply:
I'm not agree but it is up to you (feel free to close this PR).